pages/index.md

```diff
@@ -8,6 +8,8 @@ PyCon UK is the annual gathering of the UK Python community and its friends
 from around the world. This year, the conference is in Cardiff, from Thursday
 15th to Monday 19th September.
 
+[Click here](/news/20160920-impressions/) to read how the attendees felt the conference went.
```

Comment:
How about 

`Read about [how the attendees felt the conference went](/news/20160920-impressions/).`

?